process/block/mbsSelectionSession.go

```diff
@@ -126,64 +129,60 @@ func (s *miniBlocksSelectionSession) GetMiniBlockHashes() [][]byte {
 	return miniBlockHashes
 }
 
-// AddReferencedMetaBlock adds a meta block and its hash to the session
-func (s *miniBlocksSelectionSession) AddReferencedMetaBlock(metaBlock data.HeaderHandler, metaBlockHash []byte) {
+// AddReferencedHeader adds a header and its hash to the session
+func (s *miniBlocksSelectionSession) AddReferencedHeader(header data.HeaderHandler, headerHash []byte) {
 	s.mut.Lock()
 	defer s.mut.Unlock()
 
-	if check.IfNil(metaBlock) || len(metaBlockHash) == 0 {
+	if check.IfNil(header) || len(headerHash) == 0 {
```

Comment:
not related to this pr: can be done before mutex lock

Reply:
done